src/nft/components/explore/Banner.tsx

```diff
@@ -0,0 +1,195 @@
+import clsx from 'clsx'
+import { Box } from 'nft/components/Box'
+import { Center, Column, Row } from 'nft/components/Flex'
+import { VerifiedIcon } from 'nft/components/icons'
+import { bodySmall, buttonMedium, header1, section } from 'nft/css/common.css'
+import { vars } from 'nft/css/sprinkles.css'
+import { fetchTrendingCollections } from 'nft/queries'
+import { TimePeriod, TrendingCollection } from 'nft/types'
+import { formatEthPrice } from 'nft/utils/currency'
+import { putCommas } from 'nft/utils/putCommas'
+import { formatChange, toSignificant } from 'nft/utils/toSignificant'
+import { ReactNode, useEffect, useReducer, useState } from 'react'
+import { useQuery } from 'react-query'
+import { Link } from 'react-router-dom'
+
+import * as styles from './Explore.css'
```

Comment:
Do you intend to share styles across components? If not, can you rename this to `./Banner.css.ts` for consistency?

Reply:
There will be other styles here for the value prop and activity feed, but I'll rename for now.